pkg/controller/baremetalhost/baremetalhost_controller_test.go

```diff
@@ -290,8 +290,7 @@ func TestPause(t *testing.T) {
 
 	tryReconcile(t, r, host,
 		func(host *metal3v1alpha1.BareMetalHost, result reconcile.Result) bool {
-			if result.Requeue && result.RequeueAfter == pauseRetryDelay &&
-				len(host.Finalizers) == 0 {
+			if !result.Requeue {
```

Comment:
Should still check that the finalizer is not set here, shouldn't we?

Reply:
Done